src/App.tsx

```diff
@@ -6,7 +6,7 @@ import { Provider } from 'react-redux';
 import SkillComponent from './components/create-skill-component/CreateSkillComponent';
 import { SearchCurriculumComponent } from './components/search-curriculum-component/SearchCurriculumComponent';
 import PopupComponent from './components/popup-component/PopupButtonComponent';
-import CreateCategory from './components/create-category-component/CreateCategoryComponent';
+import { CreateCategoryComponent } from './components/create-category-component/CreateCategoryComponent';
```

Comment:
This is no longer getting the default export, which might break redux, even if redux isn't set up yet, you are still using a default export anyway so should follow along standards